### PR TITLE
66 Remove capital funds category enum

### LINKED ISSUE #66: Remove "capital fund category" enum
## Acceptance Criteria
- [x] Rerun drizzle kit introspection against the update zoning api database
- [ ] Remove instances where data is cast to `capital fund category` type

## Related
Depends on NYCPlanning/ae-zoning-api#391

### PR DESCRIPTION
**Pull request by horatiorosa**
#### Description 

1. removed instances where data is cast to capital fund category type
2. re-ran drizzle kit introspection (migration)

#### Tickets
Closes #66

### CODE REVIEW

File: drizzle/migration/schema.ts [hunk 1]
```diff
@@ -1,26 +1,10 @@
-import { pgEnum } from "drizzle-orm/pg-core";
+import { pgEnum } from "drizzle-orm/pg-core"
```
Comment (TangoYankee) [inline drizzle/migration/schema.ts:1]: We don't have pre-commit hooks in this repo. So, we'll need to make sure we manually run 
`npm run format`

(The lack of semi-colon makes me think we haven't run the formatter)

File: drizzle/migration/relations.ts [hunk 2]
```diff
@@ -1 +1,3 @@
-import {} from "./schema";
+import { relations } from "drizzle-orm/relations";
```
Comment (TangoYankee) [inline drizzle/migration/relations.ts:1]: We can remove this line entirely since we never use "relations"

**Review by TangoYankee — CHANGES_REQUESTED** [review]
The introspection looks good. But, we need to go through the transformation code and remove any references to the `capital_fund_category` enum.

One example in [line 72 of the capital planning tranform](https://github.com/NYCPlanning/ae-data-flow/blob/main/pg/model-transform/capital-planning.sql#L72)
As written...
```sql
'city-non-exempt'::capital_fund_category AS category
```
...should become
```sql
'city-non-exempt' AS category
``` 
(we keep the text type, rather than casting to the enum type)

File: pg/model-transform/capital-planning.sql [hunk 3]
```diff
@@ -69,8 +69,8 @@ SELECT
 	gen_random_uuid() AS id,
 	m_agency AS managing_code,
 	proj_id AS capital_project_id,
-	'city-non-exempt'::capital_fund_category AS category,
 	'adopt' AS stage,
+	'city-non-exempt' AS category,
 	adopt_ccnonexempt AS value
```
Comment (horatiorosa) [inline pg/model-transform/capital-planning.sql:74]: @TangoYankee does the order matter?

Comment (TangoYankee) [reply pg/model-transform/capital-planning.sql:74]: I think so

**Review by TangoYankee — APPROVED** [review]

**Pull request merged**